### ISSUE TITLE
Ability for format an Enum to change its format

### ISSUE THREAD

**Issue by vincentbernat** [vincentbernat/snimpy#64]
Hey

If you implement `__format__()`, you could enable a user to choose between `ok(1)`, `ok` and `1`. Think about it.

**Comment by paravoid** [comment]
Yeah, would be nice to be able to extract `ok` somehow. Right now this isn't possible at all as far as I can tell, one has to do hacks like `e.entity.enum[int(e)]`.

Have you thought about using Python's native "Enum" or "IntEnum"? That would be an even more Pythonic way :) It's 3.4+, but there is backport in the form of the enum34 package if you still care about such old versions.

**Comment by vincentbernat** [comment]
We could switch Snimpy to be 3.4+ now.
